package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "edustripe_client_v1",
+  "version": "1.0.0",
+  "description": "Edustripe client facing webapp",
+  "main": "index.js",
+  "scripts": {
+    "start": "export DEBUG=prod && node server/app.js",
+    "dev": "webpack-dev-server --open --config webpack/webpack.config.js --env.env=dev",
+    "build": "webpack --config webpack/webpack.config.js --env.env=prod"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/edustripetech/edustripe_client_v1.git"
+  },
+  "keywords": [
+    "Edtech",
+    "Edustripe",
+    "School",
+    "Education",
+    "Loan",
+    "Payment"
+  ],
+  "author": "Edustripetech",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/edustripetech/edustripe_client_v1/issues"
+  },
+  "homepage": "https://github.com/edustripetech/edustripe_client_v1#readme",
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-react": "^7.9.4",
+    "babel-loader": "^8.1.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^3.4.2",
+    "file-loader": "^6.0.0",
+    "html-webpack-plugin": "^4.0.3",
+    "node-sass": "^4.13.1",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "webpack": "^4.42.1",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.10.3",
+    "webpack-merge": "^4.2.2"
+  },
+  "dependencies": {
+    "core-util-is": "^1.0.2",
+    "cors": "^2.8.5",
+    "debug": "^4.1.1",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
```

Comment:
Really great job @obayomi96. But I have a suggestion here. Rather than building an express server, how about we just use the build command and create static files and serve them with the "serve" npm module just to make the app "portable". We can host it on any platform that way without having to manage an express app manually. What do you think?

Reply:
This is a good suggestion @chuxmykel . That way we can limit the packages installed and keep things simple. Would implement this feedback ASAP.

Reply:
All right @obayomi96. Nice work once again.